.github/workflows/e2e-aws-custom.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           github-token: ${{ secrets.GH_PERSONAL_ACCESS_TOKEN }}
           ec2-image-id: ami-01a89eee1adde309c
           ec2-instance-type: ${{ github.event.inputs.instance_type }}
-          subnet-id: subnet-02d230cffd9385bd4
+          subnet-id: subnet-024298cefa3bedd61
```

Comment:
So this changes the zone from 2a to 2b. I just checked in the console and from what I can easily tell, this does make the g6e instances available. I don't know anything about the github action setup but this change to the subnet ID seems to be correct.